src/main/scala/com/databricks/labs/overwatch/validation/DeploymentValidation.scala

```diff
@@ -100,52 +100,61 @@ object DeploymentValidation extends SparkSessionWrapper {
 
   private def validateMountCount(conf: MultiWorkspaceConfig): DeploymentValidationReport = {
 
-    val testDetails =
-      s"""WorkSpaceMountTest
-         |APIURL:${conf.api_url}
-         |DBPATWorkspaceScope:${conf.secret_scope}
-         |SecretKey_DBPAT:${conf.secret_key_dbpat}""".stripMargin
-    try {
-      val patToken = dbutils.secrets.get(scope = conf.secret_scope, key = conf.secret_key_dbpat)
-      val apiEnv = ApiEnv(false, conf.api_url, patToken, getClass.getPackage.getImplementationVersion)
-      val endPoint = "dbfs/search-mounts"
-      val mountCount = ApiCallV2(apiEnv, endPoint).execute().asDF().count()
-      if(mountCount<50)
-        {
+    if (conf.cloud.toLowerCase == "azure" &&  conf.workspace_id.trim != spark.conf.get("spark.databricks.clusterUsageTags.clusterOwnerOrgId")) {
```

Comment:
can we use the getOrgID func instead of duplicating this critical logic?

Reply:
done